app/models/post.js

```diff
@@ -74,16 +79,14 @@ exports.addModel = function(database) {
 
       that.validateOnCreate()
         .then(function(post) {
-          return Promise.all([
-            database.hmsetAsync(mkKey(['post', post.id]),
-                                { 'body': post.body,
-                                  'userId': post.userId,
-                                  'createdAt': post.createdAt.toString(),
-                                  'updatedAt': post.updatedAt.toString()
-                                }),
-            models.Timeline.newPost(post.id, that.timelineIds)
```

Comment:
Gotcha! I believe that Group#create, User#create could be affected by this as well, e.g. in User#create we create Stats object in the same Promise.all where we create user-key in the Redis.